src/app/clarity/descriptions/loadDescriptions.ts

```diff
@@ -27,50 +27,63 @@ const fetchClarity = async <T>(type: keyof typeof urls) => {
   return json;
 };
 
-const loadClarityDescriptions = dedupePromise(async (loadFromIndexedDB) => {
-  const savedVersion = Number(localStorage.getItem('clarityDescriptionVersion') ?? '0');
+const fetchRemoteDescriptions = async (version: number) => {
+  const descriptions = await fetchClarity<ClarityDescription>('descriptions');
+  set('clarity-descriptions', descriptions);
+  localStorage.setItem('clarityDescriptionVersion', version.toString());
```

Comment:
Oh, one thing - we should maybe change the key for `clarityDescriptionVersion` since we previously wrote the wrong thing into it.

Reply:
I didn't consider that a problem -- we always did that before too, and the descriptions version and stats version use entirely different formats anyway.